pkg/acs/calacs/include/acs.h

```diff
@@ -13,6 +13,9 @@
 # define ACS_FITS_REC       82
 # define SZ_STRKWVAL        68
 
+#define ACS_WFC_N_COLUMNS_PER_CHIP_INCL_OVERSCAN 4144
```

Comment:
Yes, this is only for aligning the image against the scalebycol data in the ``PCTETAB``, _c.f._ [here for more detail](https://github.com/jamienoss/hstcal/blob/4e2c39d8ff67007ffd86977ba9e6947d1a017cb3/pkg/acs/calacs/acscte/dopcte-gen2.c#L108-L111)
```c
// razColumnOffset is used to align the image with the column scaling (SCLBYCOL) in the PCTETAB.
        // The SCLBYCOL array is 8192 cols wide which is 2048*4 and therefore does NOT include the physical
        // overscan columns (24 for ACS WFC) like this array does for calwf3.
        // The raz format is all 4 amps side by side in CDAB order.
```
Does it look like it should work for subarrays to you?